### PR TITLE
EL-2606-SparkChartThemeColors

### PR DESCRIPTION
**Pull request by chris-brennan**
Updated to allow users to specify themes to set both barColor and trackColor.  Documentation updated to include a definition of the theme property, updated definitions of the barColor and trackColor properties, a table of available theme colors, and updated snippets.

### CODE REVIEW

File: docs/app/pages/charts/sections/spark-charts/spark-charts/snippets/app.html [hunk 1]
```diff
@@ -5,33 +5,33 @@
 
     <div class="col-md-1 col-sm-1 col-xs-1"></div>
 
-    <ux-spark [barColor]="charts[1].barColor" [trackColor]="charts[1].trackColor" [value]="charts[1].value" [barHeight]="charts[1].barHeight"
+    <ux-spark theme="chart3" [value]="charts[1].value" [barHeight]="charts[1].barHeight"
```
Comment (ashley-hunter) [inline docs/app/pages/charts/sections/spark-charts/spark-charts/snippets/app.html]: As you have now defined the theme as part of the object in the class you should reference it instead, eg:

```html
<ux-spark [theme]="charts[1].theme"
```

File: docs/app/pages/charts/sections/spark-charts/spark-charts/spark-charts.component.html [hunk 2]
```diff
@@ -64,15 +63,22 @@
                 <th>Description</th>
             </tr>
             <tr>
+                <td class="attribute">theme</td>
+                <td>string</td>
+                <td>Defines the color of the bar and the track. Defaults to the <code>chart1</code> color in the
```
Comment (ashley-hunter) [inline docs/app/pages/charts/sections/spark-charts/spark-charts/spark-charts.component.html]: Might want to add something to inform them the value they provide should be a color from our color palette

**Comment by chris-brennan** [comment]
Updated to reference the theme as part of the object.  Theme description updated as well

File: docs/app/pages/charts/sections/spark-charts/spark-charts/spark-charts.component.html [hunk 3]
```diff
@@ -64,15 +63,22 @@
                 <th>Description</th>
             </tr>
             <tr>
+                <td class="attribute">theme</td>
+                <td>string</td>
+                <td>Defines the color of the bar and the track. This should specifiy a color from our color palette. Defaults to the <code>chart1</code> color in the
```
Comment (ashley-hunter) [inline docs/app/pages/charts/sections/spark-charts/spark-charts/spark-charts.component.html]: Typo `specifiy`

**Comment by chris-brennan** [comment]
fixed typo

**Pull request merged**